script/format_sql

```diff
@@ -16,144 +16,6 @@ from bigquery_etl.format_sql.formatter import reformat  # noqa E402
-    "templates/telemetry_derived/clients_daily_histogram_aggregates_v1/query.sql",
-    "templates/telemetry_derived/clients_daily_keyed_boolean_aggregates_v1/query.sql",
-    "templates/telemetry_derived/clients_daily_keyed_histogram_aggregates_v1/query.sql",
-    "templates/telemetry_derived/clients_daily_keyed_scalar_aggregates_v1/query.sql",
-    "templates/telemetry_derived/clients_daily_scalar_aggregates_v1/init.sql",
-    "templates/telemetry_derived/clients_daily_scalar_aggregates_v1/query.sql",
-    "templates/telemetry_derived/clients_histogram_aggregates_v1/init.sql",
-    "templates/telemetry_derived/clients_histogram_aggregates_v1/query.sql",
-    "templates/telemetry_derived/clients_histogram_bucket_counts_v1/query.sql",
-    "templates/telemetry_derived/clients_histogram_probe_counts_v1/query.sql",
-    "templates/telemetry_derived/clients_scalar_aggregates_v1/init.sql",
-    "templates/telemetry_derived/clients_scalar_aggregates_v1/query.sql",
-    "templates/telemetry_derived/clients_scalar_bucket_counts_v1/query.sql",
-    "templates/telemetry_derived/clients_scalar_probe_counts_v1/query.sql",
-    "templates/telemetry_derived/core_clients_daily_v1/query.sql",
-    "templates/telemetry_derived/core_clients_last_seen_v1/init.sql",
-    "templates/telemetry_derived/core_clients_last_seen_v1/query.sql",
-    "templates/telemetry_derived/core_live/view.sql",
-    "templates/telemetry_derived/devtools_events_amplitude_v1/view.sql",
-    "templates/telemetry_derived/error_aggregates/query.sql",
-    "templates/telemetry_derived/event_events_v1/init.sql",
-    "templates/telemetry_derived/event_events_v1/query.sql",
-    "templates/telemetry_derived/events_live/view.sql",
-    "templates/telemetry_derived/experiment_enrollment_aggregates_v1/init.sql",
-    "templates/telemetry_derived/experiment_enrollment_aggregates_v1/query.sql",
-    "templates/telemetry_derived/experiments_daily_active_clients_v1/init.sql",
-    "templates/telemetry_derived/firefox_desktop_exact_mau28_by_client_count_dimensions_v1/query.sql",  # noqa E501
-    "templates/telemetry_derived/fxa_users_services_daily_v1/query.sql",
-    "templates/telemetry_derived/fxa_users_services_last_seen_v1/query.sql",
-    "templates/telemetry_derived/glam_user_counts_v1/query.sql",
-    "templates/telemetry_derived/histogram_percentiles_v1/query.sql",
-    "templates/telemetry_derived/latest_versions/query.sql",
-    "templates/telemetry_derived/main_events_v1/init.sql",
-    "templates/telemetry_derived/main_events_v1/query.sql",
-    "templates/telemetry_derived/scalar_percentiles_v1/query.sql",
-    "templates/telemetry_derived/smoot_usage_desktop_v2/query.sql",
-    "templates/telemetry_derived/smoot_usage_fxa_v2/query.sql",
-    "templates/telemetry_derived/smoot_usage_new_profiles_v2/query.sql",
-    "templates/telemetry_derived/smoot_usage_nondesktop_v2/query.sql",
-    "templates/telemetry_derived/surveygizmo_daily_attitudes/init.sql",
```

Comment:
I think we should either `s/templates/sql/` here, or remove these exceptions in a separate PR, rather include formatting changes in this PR.

Reply:
I changed it to ignoring the SQL files instead and did undo all the previous formatting of those files for this PR.